### ISSUE TITLE
apache2+wsgi install does not work in ubuntu 16.04

### ISSUE THREAD

**Issue by mostynb** [jensl/critic#113]
When selecting the apache2+wsgi option in a clean ubuntu 16.04 machine, apache fails to start critic-main.  It works on a clean ubuntu 14.04 machine, however.

Extract from logs:
[Wed Jan 25 22:58:33.446868 2017] [wsgi:warn] [pid 4372:tid 140071912109952] mod_wsgi: Compiled for Python/2.7.11.
[Wed Jan 25 22:58:33.447107 2017] [wsgi:warn] [pid 4372:tid 140071912109952] mod_wsgi: Runtime using Python/2.7.12.
[Wed Jan 25 22:58:33.447301 2017] [wsgi:debug] [pid 4372:tid 140071912109952] src/server/mod_wsgi.c(7362): mod_wsgi (pid=4372): Socket for 'critic-main' is '/var/run/apache2/wsgi.4372.0.1.sock'.
[Wed Jan 25 22:58:33.447397 2017] [wsgi:debug] [pid 4372:tid 140071912109952] src/server/mod_wsgi.c(7420): mod_wsgi (pid=4372): Listen backlog for socket '/var/run/apache2/wsgi.4372.0.1.sock' is '100'.
[Wed Jan 25 22:58:33.447962 2017] [mpm_event:notice] [pid 4372:tid 140071912109952] AH00489: Apache/2.4.18 (Ubuntu) mod_wsgi/4.3.0 Python/2.7.12 configured -- resuming normal operations
[Wed Jan 25 22:58:33.448064 2017] [mpm_event:info] [pid 4372:tid 140071912109952] AH00490: Server built: 2016-07-14T12:32:26
[Wed Jan 25 22:58:33.448150 2017] [core:notice] [pid 4372:tid 140071912109952] AH00094: Command line: '/usr/sbin/apache2'
[Wed Jan 25 22:58:33.448228 2017] [core:debug] [pid 4372:tid 140071912109952] log.c(1546): AH02639: Using SO_REUSEPORT: yes (1)
[Wed Jan 25 22:58:33.448864 2017] [wsgi:info] [pid 4378:tid 140071912109952] mod_wsgi (pid=4378): Initializing Python.
[Wed Jan 25 22:58:33.449130 2017] [wsgi:info] [pid 4377:tid 140071912109952] mod_wsgi (pid=4377): Initializing Python.
[Wed Jan 25 22:58:33.453044 2017] [wsgi:info] [pid 4376:tid 140071912109952] mod_wsgi (pid=4376): Starting process 'critic-main' with uid=112, gid=119 and threads=25.
[Wed Jan 25 22:58:33.459626 2017] [wsgi:info] [pid 4377:tid 140071912109952] mod_wsgi (pid=4377): Attach interpreter ''.
[Wed Jan 25 22:58:33.459823 2017] [wsgi:info] [pid 4376:tid 140071912109952] mod_wsgi (pid=4376): Initializing Python.
[Wed Jan 25 22:58:33.462786 2017] [wsgi:info] [pid 4375:tid 140071912109952] mod_wsgi (pid=4375): Starting process 'critic-main' with uid=112, gid=119 and threads=25.
[Wed Jan 25 22:58:33.462963 2017] [wsgi:info] [pid 4375:tid 140071912109952] mod_wsgi (pid=4375): Initializing Python.
[Wed Jan 25 22:58:33.483698 2017] [wsgi:info] [pid 4378:tid 140071912109952] mod_wsgi (pid=4378): Attach interpreter ''.
[Wed Jan 25 22:58:33.484292 2017] [mpm_event:debug] [pid 4377:tid 140071820551936] event.c(2094): AH02471: start_threads: Using epoll
[Wed Jan 25 22:58:33.485246 2017] [mpm_event:debug] [pid 4378:tid 140071820551936] event.c(2094): AH02471: start_threads: Using epoll
[Wed Jan 25 22:58:34.449453 2017] [core:notice] [pid 4372:tid 140071912109952] AH00051: child pid 4375 exit signal Segmentation fault (11), possible coredump in /etc/apache2
[Wed Jan 25 22:58:34.449557 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4375): Process 'critic-main' has died, deregister and restart it.
[Wed Jan 25 22:58:34.449589 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4375): Process 'critic-main' terminated by signal 11
[Wed Jan 25 22:58:34.449601 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4375): Process 'critic-main' has been deregistered and will no longer be monitored.
[Wed Jan 25 22:58:34.450052 2017] [core:notice] [pid 4372:tid 140071912109952] AH00051: child pid 4376 exit signal Segmentation fault (11), possible coredump in /etc/apache2
[Wed Jan 25 22:58:34.450081 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4376): Process 'critic-main' has died, deregister and restart it.
[Wed Jan 25 22:58:34.450089 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4376): Process 'critic-main' terminated by signal 11
[Wed Jan 25 22:58:34.450097 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4376): Process 'critic-main' has been deregistered and will no longer be monitored.
[Wed Jan 25 22:58:34.451263 2017] [wsgi:info] [pid 4436:tid 140071912109952] mod_wsgi (pid=4436): Starting process 'critic-main' with uid=112, gid=119 and threads=25.
[Wed Jan 25 22:58:34.451708 2017] [wsgi:info] [pid 4435:tid 140071912109952] mod_wsgi (pid=4435): Starting process 'critic-main' with uid=112, gid=119 and threads=25.
[Wed Jan 25 22:58:34.452310 2017] [wsgi:info] [pid 4436:tid 140071912109952] mod_wsgi (pid=4436): Initializing Python.
[Wed Jan 25 22:58:34.452816 2017] [wsgi:info] [pid 4435:tid 140071912109952] mod_wsgi (pid=4435): Initializing Python.
[Wed Jan 25 22:58:35.451765 2017] [core:notice] [pid 4372:tid 140071912109952] AH00051: child pid 4435 exit signal Segmentation fault (11), possible coredump in /etc/apache2
[Wed Jan 25 22:58:35.452188 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4435): Process 'critic-main' has died, deregister and restart it.
[Wed Jan 25 22:58:35.452412 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4435): Process 'critic-main' terminated by signal 11
[Wed Jan 25 22:58:35.452453 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4435): Process 'critic-main' has been deregistered and will no longer be monitored.
[Wed Jan 25 22:58:35.452918 2017] [core:notice] [pid 4372:tid 140071912109952] AH00051: child pid 4436 exit signal Segmentation fault (11), possible coredump in /etc/apache2
[Wed Jan 25 22:58:35.452948 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4436): Process 'critic-main' has died, deregister and restart it.
[Wed Jan 25 22:58:35.452956 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4436): Process 'critic-main' terminated by signal 11
[Wed Jan 25 22:58:35.452964 2017] [wsgi:info] [pid 4372:tid 140071912109952] mod_wsgi (pid=4436): Process 'critic-main' has been deregistered and will no longer be monitored.

**Comment by jensl** [comment]
Yeah, there are known issues with that configuration. I would suggest using one of the others instead.
